packages/jest-jasmine2/src/jasmine/spyRegistry.ts

```diff
@@ -39,7 +39,7 @@ const formatErrorMsg = (domain: string, usage?: string) => {
   return (msg: string) => domain + ' : ' + msg + usageDefinition;
 };
 
-function isSpy(putativeSpy: any) {
+function isSpy(putativeSpy: Spy) {
```

Comment:
this is supposed to be a typeguard, so maybe

```suggestion
function isSpy(putativeSpy: unknown): putativeSpy is Spy {
```
?